packages/mdc-button/README.md

```diff
@@ -70,6 +70,17 @@ We recommend you load [Material Icons](https://material.io/icons/) from Google F
 </button>
 ```
 
+It's also possible to use an SVG icon:
+
+```html
+<button class="mdc-button">
+  <svg class="mdc-button__icon" xmlns="http://www.w3.org/2000/svg" viewBox="...">
```

Comment:
I think the viewBox attr is pretty important. Can you add something to highlight that? If it isn't `0 0 24 24` then people need to go to the demo and figure that out.

Reply:
IIUC, viewBox is dependent on the particular SVG, which is why I specifically suggested leaving it in, but not prescribing what its values are. The SVGs provided by Material Icons might all be in a 24x24 space, but that's not necessarily true of someone's custom icons.

Reply:
Sorry - this will actually be different in every svg. Please ignore